internal/jobs/tiktok.go

```diff
@@ -55,38 +63,42 @@ func NewTikTokTranscriber(jc types.JobConfiguration, statsCollector *stats.Stats
 
 	// Get configurable values from job configuration
 	if err := jc.Unmarshal(&config); err != nil {
-		logrus.WithError(err).Debug("TikTokTranscriber: Could not unmarshal job configuration, using all defaults")
+		logrus.WithError(err).Warn("failed to unmarshal TikTokTranscriptionConfiguration from JobConfiguration, using defaults where applicable")
 	}
-
-	// Set defaults for configurable values if not provided
-	if config.DefaultLanguage == "" {
-		config.DefaultLanguage = "eng-US"
+	// Ensure Apify key aligns with Twitter's pattern (explicit getter wins)
+	config.ApifyApiKey = jc.GetString("apify_api_key", config.ApifyApiKey)
+	if config.ApifyApiKey != "" {
+		if c, err := tiktokapify.NewTikTokApifyClient(config.ApifyApiKey); err != nil {
+			logrus.Errorf("Failed to create Apify client at startup: %v", err)
+		} else if err := c.ValidateApiKey(); err != nil {
```

Comment:
If it failed the validation, we should perhaps overwrite it with `""` so capability detection works.